code/modules/mining/machine_stacking.dm

```diff
@@ -137,26 +137,26 @@
 	if (input_dir == output_dir)
 		rotate(input)
 
-/obj/machinery/mineral/stacking_machine/proc/process_sheet(obj/item/stack/sheet/inp)
-	if(QDELETED(inp))
+/obj/machinery/mineral/stacking_machine/proc/process_sheet(obj/item/stack/sheet/input)
+	if(QDELETED(input))
 		return
 
 	// Dump the sheets to the silo if attached
 	if(materials.silo && !materials.on_hold())
-		var/matlist = inp.custom_materials & materials.mat_container.materials
+		var/matlist = input.custom_materials & materials.mat_container.materials
 		if (length(matlist))
-			materials.mat_container.insert_item(inp, context = src)
+			materials.mat_container.insert_item(input, context = src)
```

Comment:
```suggestion
			materials.insert_item(input)
```

Reply:
Inserting will also delete the item so you want to move this last after processing

Reply:
all I did was rename this variable but I can touch it up while I'm in the file I suppose
edit: damn is `materials` a component we hold a direct reference to on a bunch of objects? thats gross and im sad you made me learn this 😔